client/shared/Fi.re

```diff
@@ -1,248 +1,94 @@
-/* react-icons: feather icon */
-module IconContext = {
```

Comment:
This was not used?

Reply:
Looks like this icon does not used in code.